### ISSUE TITLE
TypeError: Cannot read property 'name' of null

### ISSUE THREAD

**Issue by pedroteixeira** [skolmer/babel-plugin-i18n-tag-translate#4]
I am getting:
```
Module build failed: TypeError: Cannot read property 'name' of null
    at Object.ClassDeclaration (.../node_modules/babel-plugin-i18n-tag-translate/lib/index.js:136:127)
``` 

It seems to be caused by such sintax:
`export default class extends BaseClass`

(I did not know myself it was possible to have a nameless class :p)

**Comment by skolmer** [comment]
Can you give some more details about how you are using this lib and how the babel configuration looks like? What babel version are you using?

**Comment by pedroteixeira** [comment]
You can easily reproduce the error with something like this:

```
import React from "react";
export default class extends React.Component {
  render() {
    return null;
  }
}
``` 

The current transformer in the babel plugin is assuming it has a `name` - this case is nameless class.

It's a file not even using `i18n`, but the plugin throws an exception for this

**Comment by skolmer** [comment]
Ok, got it. I will create a test case for anonymous classes and see what I can do. Thanks for reporting!

**Comment by skolmer** [comment]
@pedroteixeira I have added a testcase for 
```js
@i18nGroup('custom group 3')
export default class { }
```

and

```js
@i18nGroup('custom group 3')
export default class extends BaseClass { }
```

Unfortunately I couldn't reproduce the issue. Are you maybe using some old babel version?

**Comment by pedroteixeira** [comment]
hmm.. I'm using Babel 7 `"@babel/core": "^7.0.0-beta.42"` with `"babel-core": "7.0.0-bridge.0",` - perhaps that's relaed? which babel version did you use?

**Comment by pedroteixeira** [comment]
btw, thank you taking the the time to reply. Really appreciated and thanks for sharing the library :)

**Comment by skolmer** [comment]
I'm using babel 6 with https://github.com/loganfsmyth/babel-plugin-transform-decorators-legacy

looks like there are changes in babel 7 in how to use class decorators: https://babeljs.io/docs/plugins/transform-decorators/

**Comment by luxcem** [comment]
The issue is still here with "@babel/cli": "^7.0.0" & "@babel/core": "^7.1.2".

**Comment by luxcem** [comment]
Maybe use `node.loc.identifier` instead of `node.id.name` in these cases?

**Comment by skolmer** [comment]
thanks, this should be fixed now, can you check v1.0.1
